vjunosevolved/README.md

```diff
@@ -0,0 +1,17 @@
+# vrnetlab / Juniper vJunosEvolved
+
+This is the vrnetlab docker image for Juniper's vJunosEvolved.
+
+> Available with [containerlab](https://containerlab.dev) as vr-vjunosevolved.
```

Comment:
```suggestion
> Available with [containerlab](https://containerlab.dev) as juniper_vjunosevolved.
```

We are drifting away from using `vr-*` kind names in favor of `vendor_nos` type of naming.